stdlib/REPL/src/LineEdit.jl

```diff
@@ -322,7 +322,7 @@ end
 
 # Show available completions
 function show_completions(s::PromptState, completions::Vector{String})
-    colmax = maximum(map(length, completions))
+    colmax = occursin('\n', first(completions)) ? width(terminal(s))-2 : maximum(map(length, completions))
     num_cols = max(div(width(terminal(s)), colmax+2), 1)
```

Comment:
```suggestion
    if any(occursin('\n'), completions)
        foreach(println, completions)
        return
    end
    colmax = maximum(length, completions) # n.b. length >= textwidth
    num_cols = max(div(width(terminal(s)), colmax+2), 1)
```

Reply:
I think it might be more reliable to check all input lines, and then entirely bypass the extra logic

Reply:
Sure! Thanks for the snippet, I took your idea with did minor edits.